src/routes/popup/swap/review.tsx

```diff
@@ -239,7 +239,7 @@ export function SwapReviewView() {
       await swapsArray.push(updatedSwapData);
 
       // Start background monitoring
-      await startSwapMonitoring();
+      await startSwapMonitoring(true);
```

Comment:
Force restarting alarms because there might be bridge txs check going on in background which are run at 2min interval so force restarting to immediately check for the DEX AO swaps if the current alarm is not currently running. Check startSwapMonitoring implementation.